frontend/dynamic-demo-plugin/tsconfig.json

```diff
@@ -6,8 +6,9 @@
     "target": "es2016",
     "jsx": "react",
     "allowJs": true,
-    "strict": true,
```

Comment:
Note: Console plugin SDK code (and any other code it references) is not compiled in TS [strict mode](https://www.typescriptlang.org/tsconfig#strict).

Once we separate between Console provided APIs (utility functions, components, etc.) in terms of type declaration vs. implementation, we can re-enable TS strict mode in the demo plugin.